UNRELEASED.md

```diff
@@ -7,6 +7,8 @@
 ### Enhancements
 
 - Updated `Filters` to only show the "More filters" button if necessary ([#2856](https://github.com/Shopify/polaris-react/pull/2856)).
+- Added utilities for parsing video duration (https://polaris.shopify.com/components/images-and-icons/video-thumbnail) ([#2725](https://github.com/Shopify/polaris-react/pull/2725))
```

Comment:
This line can be removed. It already exists in the changelog which is why your build is failing. You might need to remove this line and rebase again 😅

Reply:
I'm sorry, something went wrong with my rebase. Just fixed it :smile:

Reply:
There is another conflict in `UNRELEASED.md` 😅

Reply:
Fixed again :sweat_smile:

Reply:
I think we are very unlucky today there is again a conflict 😢

Reply:
Let me know if you want me to get this over the finish line @thayannevls

Happy to help, and sorry about all of the conflicts

Reply:
@shioyama Yes :cry: I fixed again lol

@kyledurand Do you think we can already merge this PR?

Reply:
Now the CI is failing because of `duplicate-changelog.test.js`, let me know if I can help to fix that.